eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -156,17 +156,17 @@ jobs:
           Options: --batch-mode
           MavenOptions: $(MemoryOptions) $(LoggingOptions)
 
-      - task: Maven@3
-        displayName: 'Install azure-sdk-parent'
-        inputs:
-          mavenPomFile: parent/pom.xml
-          options: '$(DefaultOptions) -DskipTests -Dgpg.skip'
-          mavenOptions: '$(MemoryOptions)  $(LoggingOptions)'
-          javaHomeOption: 'JDKVersion'
-          jdkVersionOption: '1.11'
-          jdkArchitectureOption: 'x64'
-          publishJUnitResults: false
-          goals: 'install'
+      # - task: Maven@3
```

Comment:
This doesn't run aggregates or anything, was this removed since parent is less useful given @JimSuplizio's version management changes?

Reply:
It seemed redundant to me. Do we need to install it? Given we find the parent POMs via links (or via Maven Central) won't they end up getting installed anyway?

Reply:
@alzimmermsft the parent still contains some plugin definitions. It's unclear if we can simply remove it but maybe we should look at if we can. I'd love to have less 'stuff' in our repo.

Reply:
Ah, didn't realize either of these!